testserver/testserver.go

```diff
@@ -417,22 +418,24 @@ func (ts *testServerImpl) pollListeningURLFile() error {
 		state := ts.state
 		ts.mu.Unlock()
 		if state != stateRunning {
-			return fmt.Errorf("server stopped or crashed before listening URL file was available")
+			return fmt.Errorf("testserver: server stopped or crashed before listening" +
+				" URL file was available")
 		}
 
 		var err error
 		data, err = ioutil.ReadFile(ts.listeningURLFile)
 		if err == nil {
 			break
 		} else if !os.IsNotExist(err) {
-			return fmt.Errorf("unexpected error while reading listening URL file: %v", err)
+			return fmt.Errorf("testserver: unexpected error while reading listening URL"+
```

Comment:
we should wrap the error here rather than use fmt.Errorf

see https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/1676640931/Error+handling+basics
(although for this library we cannot use cockroachdb/errors, so we can use the golang errors package